alg/gdaltransformer.cpp

```diff
@@ -467,51 +469,56 @@ GDALSuggestedWarpOutput2( GDALDatasetH hSrcDS,
         if( nSteps > 20 )
         {
             nSteps = 20;
+            nStepsPlusOne = 21;
             goto retry;
         }
         return CE_Failure;
     }
 
     padfY = padfX + nSampleMax;
-    padfZ = padfX + nSampleMax * 2;
+    padfZ = padfX + (nSampleMax<<1);
```

Comment:
unnecessary/hard-to-read change. compilers are smart enough to optimize a multiplication by two with a bitshift. no need to make humans aware of that trick